servers/todo/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "perf-todo",
+  "version": "1.0.0",
+  "description": "Todo API reference app for Express performance benchmarking",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "dev": "nodemon server.js"
```

```suggestion
    "dev": "node --watch server.js"
```

We can remove one dependency :)

---

applied in my updated.